src/olympia/search/tests/test_search_ranking.py

```diff
@@ -709,7 +709,7 @@ def test_scenario_tabbicat(self):
         self._check_scenario(
             'tabbicat',
             (
-                ['Tabby Cat', 3255],
```

Comment:
That was the only result which had its score affected by this change, because it's not matching as well as before. Still high enough to be considered the best result for that query in our test db though.